ROADMAP.md

```diff
@@ -25,23 +25,43 @@ Git LFS. If you have an idea for a new feature, open an issue for discussion.
 | :soon: | Teach `git lfs track` and others to warn when `git lfs install` hasn't been run (or auto-install) | [#1167](https://github.com/github/git-lfs/issues/1167) |
+| | Investigate hanging pushes/pulls when git credential helper is not set | [#197](https://github.com/github/git-lfs/issues/197) |
+| | Support git ssh shorthands | [#278](https://github.com/github/git-lfs/issues/278) |
+| | Support `GIT_CONFIG` | [#318](https://github.com/github/git-lfs/issues/318) |
+| | Warn when Git version is unsupported | [#410](https://github.com/github/git-lfs/issues/410) |
+| | Detect when credential cacher is not setup | [#523](https://github.com/github/git-lfs/issues/523) |
+| | Fix error logging from `git clone` errors | [#513](https://github.com/github/git-lfs/issues/513) |
+| | Investigate cherry picking issues | [#438](https://github.com/github/git-lfs/issues/438) |
+| | dynamic blob size cutoff for pointers | [#524](https://github.com/github/git-lfs/issues/524) |
+| | windows `--help` support | [#394](https://github.com/github/git-lfs/issues/394) |
+| | Investigate git hook installs within git worktree | [#1385](https://github.com/github/git-lfs/issues/1385) |
+| | Support ssh username in ssh config | [#754](https://github.com/github/git-lfs/issues/754) |
+| | Investigate `autocrlf` for lfs objects | [#723](https://github.com/github/git-lfs/issues/723) |
 
 ## Upcoming Features
 
 | | Name | Ref |
 | ------ | ---- | --- |
-| :construction: | File locking [#666](https://github.com/github/git-lfs/pull/666) | |
-| :ship: | Resumable uploads and downloads [#414](https://github.com/github/git-lfs/issues/414) | |
+| :construction: | File locking | [#666](https://github.com/github/git-lfs/pull/666) |
+| :ship: | Resumable uploads and downloads | [#414](https://github.com/github/git-lfs/issues/414) |
 | :construction: | Wrapped versions of `git pull` & `git checkout` that optimize without filters like `git lfs clone` | |
 | | Remove non-batch API route in client | |
 
 ## Possible Features
 
 | | Name | Ref |
 | ------ | ---- | --- |
+| | Support tracking files by size | [#282](https://github.com/github/git-lfs/issues/282)
 | | Binary diffing - reduce the amount of content sent over the wire. | |
 | | Client side metrics reporting, so the Git LFS server can optionally track how clients are performing. | |
 | | Pure SSH: full API & transfer support for SSH without redirect to HTTP | |
+| | Compression of files in `.git/lfs/objects` | [#260](https://github.com/github/git-lfs/issues/260) |
+| | LFS Migration tool | [#326](https://github.com/github/git-lfs/issues/326) |
+| | Automatic upgrades | [#531](https://github.com/gihtub/git-lfs/issues/531) |
+| | Investigate `git add` hash caching | [#574](https://github.com/github/git-lfs/issues/574) |
+| | `git lfs archive` command | [#1322](https://github.com/github/git-lfs/issues/1322) |
+| | Support 507 http responses | [#1327](https://github.com/github/git-lfs/issues/1327) |
```

Comment:
I have a couple of questions about HTTP 507 support.

- Does adding support involve anything more than updating the list of optional status codes to mention it?  If not, I can send a PR to make that update.

- Is it going to mean a new API version?

Reason I ask about the API version is that I'd like to [include support for this in JGit](https://git.eclipse.org/r/#/c/79315/) (which currently implements Git LFS v1 Batch API) but if it's in a new API version (likely also including other stuff) it might not be as easy.